### PR TITLE
Youtube Clone

### PR DESCRIPTION
**Pull request by 0prodigy**
The very first pull request

### CODE REVIEW

**Comment by theshauryasingh** [comment]
the two tags header and nav both weren't required. One of them could have been skipped

**Comment by Dandush03** [comment]
### :red_circle: Status Require: changes
# Hey @0prodigy and @theshauryasingh :wave: 
### Good attempts on this one :muscle: :+1: 
Here are the issues to address to get the project meet the requirement:wink:.

- [ ] The login button content should be blue the same as the original page.
 #### current: 
Screenshot from 2020-03-24 07-54-34
 #### expected:
Screenshot from 2020-03-24 07-53-21

- [ ] The element in your comments section is not well aligned as the original page
 #### current: 
Screenshot from 2020-03-24 07-48-33
 #### expected:
Screenshot from 2020-03-24 07-58-26




## [_Optional_]
- [ ] Implement the use of  the [`<nav>` tag](http://html5doctor.com/nav-element/) for the major navigation links
- [ ] Implement the use of the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document

##### Kindly address all these changes and submit a new code review
### Happy coding:computer: 
#### Daniel Laloush

**Comment by sumancrest0001** [comment]
You guys have done a great job. Now, you can merge the branch to master. Best of luck with the next project. 
mission

File: style.css [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+
+.left {
+  margin: 3rem;
+  width: 50%;
+}
+
+.left .video {
+  height: 360px;
+  width: 660px;
+}
```
Comment (sumancrest0001) [inline style.css]: [OPTIONAL] The Alignment of the video and description section doesn't look like that of original YouTube website. you can change the width to 640px to ensure proper alignment of elements in the main container.

**Pull request merged**